_layouts/post.html

```diff
@@ -79,6 +79,11 @@ <h1 class="title">{{ page.title }}</h1>
         <i class="fa fa-{{ social.icon | remove_first: '-square' }}-square fa-lg"></i>
       </a>
     {% endif %}
+	{% if social.name == "Reddit" and social.share == true %}
+		<a href="//www.reddit.com/submit" onclick="window.location = '//www.reddit.com/submit?url=' + encodeURIComponent('{{ full_url }}') + '&title={{page.title}}'; return false">
+			<i class="fa fa-{{ social.icon | remove_first: '-square' }}-square fa-lg"></i>
+		</a>
+	{% endif %}
```

Comment:
I'm using WebStorm, which is adhering to the .editorconfig options in the repository for html documents.
~~~
[*.{html,js,scss}]
charset = utf-8
indent_style = tab
indent_size = 2
~~~
However, the rest of the file is using spaces for indentation. I'm going to assume that GitHub uses tab widths of 4, causing the above diff to look out of whack. The below screenshot shows the different whitespace characters.
<img width="608" alt="screen shot 2015-12-24 at 1 24 43 pm" src="https://cloud.githubusercontent.com/assets/1518405/11988316/c900309c-aa41-11e5-921d-416ea2e8a1bd.png">